### DISCUSSION TITLE
native type timestamptz not support in 2.11.0?

### DISCUSSION THREAD

**Discussion by michael-land** [prisma/prisma#4215] [Q&A] [answered]
I tried 
```
@db.Timestamptz
@db.TimestampTZ
@db.TimestampTz
```

None of them works

**Comment by pantharshit00** [comment, marked as answer]
@xiaoyu-tamu Timestampz is currently not supported in the first release of native types. You will need to wait here.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
